### PR TITLE
Frontend Framework

### PR DESCRIPTION
**Pull request by frankfka**
# Pull Request

## Description
React front-end framework - everything is laid out but just missing the fetch calls to retrieve the generated text.

## Testing
run yarn start within the 'frontend' folder

### CODE REVIEW

File: frontend/public/manifest.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
```
Comment (rwblickhan) [inline frontend/public/manifest.json]: These should probably be made specific to Mimic, right?

File: frontend/src/App.test.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+
+it('renders without crashing', () => {
```
Comment (rwblickhan) [inline frontend/src/App.test.js]: 👍 

We should consider running this as part of our Travis builds.

File: frontend/src/index.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+import './index.css';
+import App from './App';
+import * as serviceWorker from './serviceWorker';
+
+ReactDOM.render(<App />, document.getElementById('root'));
+
+// If you want your app to work offline and load faster, you can change
+// unregister() to register() below. Note this comes with some pitfalls.
+// Learn more about service workers: https://bit.ly/CRA-PWA
+serviceWorker.unregister();
```
Comment (rwblickhan) [inline frontend/src/index.js]: Do we need all the service worker code?

File: frontend/public/manifest.json (hunk 1 above)
Comment (frankfka) [reply frontend/public/manifest.json]: Haha true. Fixed! We also should have a favicon/logo

File: frontend/src/index.js (hunk 3 above)
Comment (frankfka) [reply frontend/src/index.js]: Nope, removed

**Comment by frankfka** [comment]
> I tried running it and it looks great! I'm probably not the best person to review this (what's frontend code lol) but I did leave a few comments - I don't mind approving if you think it's ready, or we can wait till @Annnnnieee finishes the server so you can contact the actual endpoints.

Yeah I can wait until the endpoints are complete

File: frontend/public/manifest.json (hunk 1 above)
Comment (rwblickhan) [reply frontend/public/manifest.json]: Hmm, might be a bit late, but we could ask design for one 😅

**Pull request merged**